keyboards/alt67/alt67.h

```diff
@@ -0,0 +1,23 @@
+#ifndef _ALT67_H_
+#define _ALT67_H_
+
+#include "samd51j18a.h"
+#include "config.h"
+#include "config_led.h"
+#include "matrix.h"
+
+#define KEYMAP( \
```

Comment:
Please replace the `KEYMAP` with `LAYOUT`.

Also, if this matches a community layout, that should be used here (eg, `LAYER_xxx`)